### ISSUE TITLE
Formula for level 80 applied according to crafter level, should be recipe level.

### ISSUE THREAD

**Issue by Inzoum** [doxxx/ffxiv-craft-opt-web#304]
## Required Information

Class: Culinarian
Level: 80
Craftsmanship: 2283
Control: 2139
CP: 536
Recipe Name: Blood Bouillabaisse
Recipe Level: 79
Solver Seed: 1564836457396

## Expected Behaviour

As this is a level 79 recipe, I’m expecting values comparable to those I had at level 79 CUL. The difficulty of the recipe hasn’t changed in-game and leveling to 80 has only made it easier due to higher stats.

## Actual Behaviour

The formula for level 80 is applied, resulting in very very low progress and quality improvements in the simulator, which are not in-line with the reality in-game. This recipe should still use the formula for level 71+ recipes. I’m expecting this occurs because my crafter level is set to 80 in the simulator. Setting the crafter level to 79 while keeping all crafting stats identical results in values matching the ones appearing in-game.

## Steps To Reproduce

Pick a level 71-79 recipe in the simulator, enter crafting stats and set level to 79, observe results, then change crafter level to 80, observe results. The values are different, indicating different formulas are used. The formulas differ based on the crafter level, when it should be dictated by the recipe level.

**Comment by Inzoum** [comment]
Actually, I get the feeling only the starred recipes are impacted by the change in formulae...

**Comment by Inzoum** [comment]
For added info: with the stats mentionned above, I get these values for 100% efficiency on Blood Bouillabaisse (L.79) at level 80 CUL:

Expected Progress: 555 (Simulator says 410)
Expected Quality: 821 (Simulator says 354)

Switching my level in the simulator to 79 yields the following results, which I’m unable to compare with in-game values as I’ve already exceeded level 79:
Progress: 512
Quality: 825

While these are a little off compared to actual in-game values for level 80, they’re more consistent. On the other hand, checking a level 80 recipe in-game with the same stats (CUL80), I get these values:

Lemonade (80):
Progress: 373 (Simulator says 373 - OK!)
Quality: 405 (Simulator says 354 - Little low, but same ballpark)

This leads me to believe that the formula for level 80 isn’t too inaccurate, it’s just applied when it shouldn’t

**Comment by doxxx** [comment]
The progress/quality gains have always been based on the crafter level. The difference between crafter and recipe level do affect it, but crafter level is the main one. However, there is a different formula used at level 80 as compared to level 71-79.

Calling @ShammyLevva ... :)

**Comment by ShammyLevva** [comment]
Yes I noted there was a significant variance for less than 80 recipes when crafter is level 80. I’ve been focused on getting crafters to 80 rather than testing this discrepancy. I’ll investigate later. 

Note though I’m helping host a major event this coming week so my time online will be very limited. Thus it may be worthwhile others testing this variance to come up with a fix until I can look at it which may not be until after the 11th.

**Comment by Enzemo** [comment]
I can chime for 80 GSM, not sure if it's related to this issue. 

Recipe is level 68, Arkhi Brewing Set.
Progress: 721
Efficiency: 783
Simulated Progress: 523
Simulated Efficiency: 336

Needless to say, it's causing me a lot of problems

**Comment by doxxx** [comment]
Thanks to @yls888 and @SaltFish-X, we have a new set of formulas for progress and quality increase. These changes are available on the beta site (ffxiv-beta.lokyst.net). Please let me know if these discrepancies are resolved.
